### DISCUSSION TITLE
Add option to display legacy Beatmap Title Text in-game.

### DISCUSSION THREAD

**Discussion by fluxie31** [ppy/osu#25219] [Ideas]
This is a feature that is in stable and not actively planned on being added to osu!lazer according to https://github.com/ppy/osu-web/issues/10643#issuecomment-1765924814. I believe that it should be added as some maps used it in really cute ways, which can still be viewed to this day on osu!stable. Examples include [Bad Apple!! (Chiptune ver.) by Sushi](https://osu.ppy.sh/beatmapsets/28222#osu/94187) and [Talent Shredder by val0108 and Lesjuh](https://osu.ppy.sh/beatmapsets/47710#osu/178966). I even requested title text changes in my own beatmaps back in 2021.

I think if it was brought back, its functionality could be extended and publicized, possibly by including an option on the osu! website for mappers to edit title text, allowing setting fonts, colors, and timing of the text showing up in-game, or even allowing creators to disable it entirely.

Additionally, the Ranking criteria could be amended to set rules and guidelines to prevent abuse of the text. I know this isn't under the development team's control, but I still think it is a good point to add.

Right now, the title text can't even be changed by NAT for stable because its functionality was limited to old.ppy.sh, which was recently made "Read-Only," preventing NAT from being able to change it.

Overall, I think a map's title text is a part of the storyboard and should be better customizable, both for legacy purposes and lazer purposes.

**Comment by peppy** [comment]
I'd recommend customising this by adding it to your storyboard directly.

See my response [here](https://github.com/ppy/osu-web/issues/10643#issuecomment-1765924814) as well.
